src/Questions/questions.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Controller } from '@nestjs/common';
```

Comment:
create 'module' folder , put Question and User into it , also change User->user (folder name)